pkg/collector/corechecks/system/wincrashdetect/probe/crashparse_test.go

```diff
@@ -51,12 +51,11 @@ func TestCrashParser(t *testing.T) {
 		FileName: "testdata/crashsample1.txt",
 	}
 	// first read in the sample data
-
-	readfn = testCrashReader
+	OverrideCrashDumpReader(testCrashReader)
 
 	parseCrashDump(wcs)
 
-	assert.True(t, wcs.Success)
+	assert.Equal(t, wcs.StatusCode, WinCrashStatusCodeSuccess)
```

Comment:
you might want to reverse the 2nd and 3rd args.  The 2nd arg is the `expected`; as written if the test were to fail here, it will say "expected <status code> got success" rather than the other way around.